bus-mapping/src/evm/opcodes/create.rs

```diff
@@ -28,20 +28,36 @@ impl<const IS_CREATE2: bool> Opcode for DummyCreate<IS_CREATE2> {
 
         let mut exec_step = state.new_step(geth_step)?;
 
+        let n_pop = if IS_CREATE2 { 4 } else { 3 };
```

Comment:
Couldn't post a comment on the correct line since that's not a change you added...

Reply:
I'm not really implementing it here. I needed to do this so that the DummyGadget constraints will pass. The constraints checks that 4 or 3 words are popped from the stack and that 1 is pushed.